nuitka/utils/WindowsResources.py

```diff
@@ -164,7 +165,7 @@ def _openFileWindowsResources(filename):
 
     BeginUpdateResource.restype = ctypes.wintypes.HANDLE
 
-    update_handle = BeginUpdateResource(filename, False)
+    update_handle = BeginUpdateResource(absFilename, False)
```

Comment:
Just a minor style issue, can you name it `fullpath` as that is what I used more often for this kind of variable in other codes in Nuitka.